shared/fs/send-link-to-chat/container.js

```diff
@@ -115,14 +115,21 @@ const mergeProps = (stateProps, {onCancel, _send, _selectChannel}, ownProps) =>
     ],
     []
   )
+
   return {
     conversation: {
       channels,
       name: elems[2],
       selectChannel: convID => _selectChannel(convID),
-      selectedChannelname: (
-        channels.find(({convID}) => convID === stateProps._sendLinkToChat.convID) || {channelname: null}
-      ).channelname,
+      selectedChannel:
```

Comment:
So if I have a folder with someone but not a conversation, the 'Send in conversation' button is grayed out without explanation. I think ideally it could create a conv and send the message into it (the "reply privately" case in `createConversation` chat actions is similar).

Reply:
Sounds good and thanks for the example! I'll do that in a separate PR.